paprika/tleap.py

```diff
@@ -112,6 +113,14 @@ class System(object):
 
     """
 
+    class ConversionToolkit(Enum):
+        """
+        An enumeration of the different toolkits for converting AMBER files to other MD formats.
+        """
+
+        ParmEd = "parmed"
+        InterMol = "intermol"
```

Comment:
Not a strong opinion, but unless there's a compelling reason that each System should own a copy of this enum, it may be better to define it at the module level.

Reply:
Good point!